providers.csv

```diff
@@ -15,3 +15,4 @@ BlueLA,f3c5a65d-fd85-463e-9564-fc95ea473f7d,https://www.bluela.com,https://api.b
 Bolt,3291c288-c9c8-42f1-bc3e-8502b077cd7f,https://www.micromobility.com/,https://bolt.miami/bolt2/api/mds,
 CLEVR,daecbe87-a9f2-4a5a-b5df-8e3e14180513,https://clevrmobility.com,https://portal.clevrmobility.com/api/la/,https://portal.clevrmobility.com/api/gbfs/en/discovery/
 Sherpa,3c95765d-4da6-41c6-b61e-1954472ec6c9,,https://mds.bird.co,https://mds.bird.co/gbfs/sherpa
+VOI,1f129e3f-158f-4df5-af9c-a80de238e334,https://www.voiscooters.com/,https://mds.voiapp.io,https://api.voiapp.io/v1/gbfs
```

Comment:
A nice random uuid

Reply:
Looks random to me

Reply:
TODO: 
`https://api.voiapp.io/v1/gbfs` => `https://mds.voiapp.io/v1/gbfs`